src/js/lib/monster.util.js

```diff
@@ -263,6 +263,13 @@ define(function(require) {
 			return isSuperDuper;
 		},
 
+		// Validate if the given port authority ID matches with the current account ID
+		isPortAuthority: function(portAuthorityID, pAccount) {
```

Comment:
This shouldn't a global util as it is specific to port request, which are handled by the porting app. Please keep it contained in the relevant port common control.